static/app/views/explore/logs/utils.tsx

```diff
@@ -229,3 +232,10 @@ export function logsPickableDays(organization: Organization): PickableDays {
     }),
   };
 }
+
+export function getDynamicLogsNextFetchThreshold(lastPageLength: number) {
+  if (lastPageLength > LOGS_GRID_SCROLL_MIN_ITEM_THRESHOLD / 0.75) {
+    return Math.floor(lastPageLength * 0.75); // Can be up to 750 on large pages.
```

Comment:
The asymmetry between the condition and the return value here threw me off for a bit here

```suggestion
  if (lastPageLength * 0.75> LOGS_GRID_SCROLL_MIN_ITEM_THRESHOLD) {
    return Math.floor(lastPageLength * 0.75); // Can be up to 750 on large pages.
```